src/Request/ConsoleRequest.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+namespace Bugsnag\BugsnagLaravel\Request;
+
+use Bugsnag\Request\RequestInterface;
+
+class ConsoleRequest implements RequestInterface
+{
+    /**
+     * The unformated console command.
+     *
+     * @var array
+     */
+    protected $command;
+
+    /**
+     * Create a new console request instance.
+     *
+     * @return void
+     */
+    public function __construct()
+    {
+        $this->command = $_SERVER['argv'];
```

Comment:
In terms of generalising it as laravel (& symfony) uses it's own resolver object by default I think it'll be more appropriate to follow the existing pattern and make Laravel/Symfony/PHP specific console handlers with a look to generalising further down the line at some point